Payload/bleemsync/etc/bleemsync/FUNC/0030_retroarch.funcs

```diff
@@ -73,7 +73,7 @@ exit_checkSaveState(){
     
   ra_last_game=`ls -t "${ra_savestate_path}/"*.auto | head -1`
 
-  if [ "$ra_last_game" == "" ]; then
+  if [ x"$ra_last_game" = x"" ]; then
```

Comment:
There's something about empty arguments that I forgot, but good practice or something to have a guard there.